libcudacxx/docs/overview.md

```diff
@@ -1,50 +1,71 @@
-  [GitHub] and included in the NVIDIA HPC SDK and CUDA Toolkit.
-If you have one of those SDKs installed, no additional installation or compiler
-  flags are needed to use libcu++.
+> [!NOTE]
+> libcu++ does not provide its own documentation for Standard Library features. 
+> Instead, libcu++ [documents which Standard Library headers](https://nvidia.github.io/cccl/libcudacxx/standard_api.html) are made available, and defers documentation of individual features within those headers to other sources like [cppreference](https://en.cppreference.com/w/).
+
+## C++ Standard Library Extensions
+
+libcu++ provides CUDA C++ developers with familiar Standard Library utilties to improve productivity and flatten the learning curve of learning CUDA.
+However, there are many aspects of writing high-performance CUDA C++ code that cannot be expressed through purely Standard conforming APIs.
+For these cases, libcu++ also provides _extensions_ of Standard Library utilities. 
+
+For example, libcu++ extends `atomic<T>` and other synchornization primitives with the notion of a "thread scope" that controls the strength of the memory fence. 
+
+To use utilities that are extensions to Standard Library features, drop the `std`:
+```cuda
+#include <cuda/atomic>
+cuda::atomic<int, cuda::thread_scope_device> x;
+```
+
+See the [Extended API](extended_api.md) section for more information. 
+
+## Fundamental CUDA-specific Abstractions
+
+Some abstractions that libcu++ provide have no equivalent in the C++ Standard Library, but are otherwise abstractions fundamental to the CUDA C++ programming model.
+
+For example, [`cuda::memcpy_async`](extended_api/asynchronous_operations/memcpy_async.md) is a vital abstraction for asynchronous data movement between global and shared memory.
+This abstracts hardware features such as `LDGSTS` on Ampere, and the Tensor Memory Accelerator (TMA) on Hopper. 
 
-## `cuda::` and `cuda::std::`
+See the [Extended API](extended_api.md) section for more information. 
 
-When used with NVCC, NVIDIA C++ Standard Library facilities live in their own
-  header hierarchy and namespace with the same structure as, but distinct from,
-  the host compiler's Standard Library:
+## Summary: `std::`, `cuda::` and `cuda::std::`
 
-* `std::`/`<*>`: When using NVCC, this is your host compiler's Standard Library
-      that works in `__host__` code only, although you can use the
-      `--expt-relaxed-constexpr` flag to use any `constexpr` functions in
-      `__device__` code.
-    With NVCC, libcu++ does not replace or interfere with host compiler's
-      Standard Library.
+* `std::`/`<*>`: This is your host compiler's Standard Library that works in `__host__` code only, although you can use the `--expt-relaxed-constexpr` flag to use any `constexpr` functions in`__device__` code. libcu++ does not replace or interfere with host compiler's Standard Library.
 * `cuda::std::`/`<cuda/std/*>`: Strictly conforming implementations of
```

Comment:
Can we drop the `Strictly` here. Technically our backports are not conforming

Reply:
Well, if you want to bring "technically" into this, the publication of a new standard obsoletes the previous one, so _technically_ they _are_ conforming.